src/pages/Account/Components/CodeSnippet.tsx

```diff
@@ -155,6 +205,27 @@ export function Code({bytecode}: {bytecode: string}) {
             Code
           </Typography>
           <StyledLearnMoreTooltip text="Please be aware that this code was provided by the owner and it could be different to the real code on blockchain. We can not not verify it." />
```

Comment:
Might want to change this based on the verification button.

Reply:
I changed the text like below

Please be aware that this code was provided by the owner and it could be different to the real code on blockchain. Please click the 'Verify Source Code' button if you want to confirm that the provided source code matches the actual code on the blockchain.